tests/moe/test_w4a8_moe_reduce.py

```
@@ -0,0 +1,75 @@
+
+import pytest
+import torch
+
+try:
+    from flashinfer.fused_moe.cute_dsl.moe_reduce_triton import (
+        build_reduce_index,
+        moe_reduce,
+    )
+
+    _OK = True
+except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)